src/components/ui/dotteddivider.tsx

```
@@ -0,0 +1,48 @@
+
+  return (
+    <div
+      className={`w-full ${colorClass} ${className}`}
+      style={{ height: `${height}px` }}
+    >
+      <svg
+        width="100%"
+        height="100%"
+        xmlns="http://www.w3.org/2000/svg"
+        style={{ display: "block" }}
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)